src/wm/mod.rs

```diff
@@ -132,6 +152,11 @@ impl FromStr for WikidataQid {
 ///
 /// assert!(WikipediaTitleNorm::from_url("https://en.wikipedia.org/not_a_wiki_page").is_err());
 /// assert!(WikipediaTitleNorm::from_url("https://wikidata.org/wiki/Q12345").is_err());
+///
+/// assert!(
+///     WikipediaTitleNorm::from_url("https://de.wikipedia.org/wiki/Breil/Brigels").unwrap() !=
```

Comment:
Can / be percent-escaped in such cases? How the generator handles it now?

Reply:
I guess it could be, I haven't looked for that. Wikipedia works with either.

See below for more details, but the generator should decode those before dumping the urls.

It looks like a handful of encoded titles still slip through, but none with `%2F`=`/`.
I made an issue with some notes about this in #7.

From my read of when it [first adds a wikipedia tag](https://github.com/organicmaps/organicmaps/blob/34bbdf6a2f077b3d629b3f17e8e05bd18a4e4110/generator/osm2meta.cpp#L241) and later [writes it as a url](https://github.com/organicmaps/organicmaps/blob/34bbdf6a2f077b3d629b3f17e8e05bd18a4e4110/indexer/feature_meta.cpp#L19):
1. If the tag looks like a url instead of the expected `lang:Article Title` format, take what's after `.wikipedia.org/wiki/`, url decode it, replace underscores with spaces, then concat that with the lang at the beginning of the url and store it.
2. Otherwise attempt to check if it's a url, replace underscores with spaces, and store it.
3. To transform it back into a url, replace spaces with underscores in the title, escape any `%`s, and add it to the end of `https://lang.wikipedia.org/wiki/`.

Glancing at the [url decoding](https://github.com/organicmaps/organicmaps/blob/34bbdf6a2f077b3d629b3f17e8e05bd18a4e4110/coding/url.cpp#L118), I don't think there's anything wrong with it - it should handle arbitrary characters, although neither the encoding or decoding look unicode-aware.